x-pack/plugins/cases/public/components/user_action_tree/user_action_markdown.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiButtonEmpty, EuiButton } from '@elastic/eui';
+import React, { useCallback } from 'react';
+import styled from 'styled-components';
+
+import * as i18n from '../case_view/translations';
+import { Form, useForm, UseField } from '../../common/shared_imports';
+import { schema, Content } from './schema';
+import { MarkdownRenderer, MarkdownEditorForm } from '../../components/markdown_editor';
+
+// TODO: Need to migrate user_action_markdown.test.tsx still.
```

Comment:
you didn't because of timeline? you probably do but plz make sure you have a list of all this stuff so we dont forget. im going to delete it from `security_solution` in my next PR so you'll have to get it off of `master`